Source/objects.cpp

```diff
@@ -2428,55 +2428,45 @@ void ObjL2Special(int x1, int y1, int x2, int y2)
 	}
 }
 
-void DoorSet(int oi, int dx, int dy)
+void __fastcall DoorSet(int oi, int dx, int dy)

```

Comment:
The `__fastcall` needs to go. We're using `__slowcall` now ;P

Reply:
oops I missed that, that's what happens when you copy/paste from beta :D

Reply:
No worries. I've literally done the same thing when ripping code many times!